src/components/footer/footer.js

```diff
@@ -1,15 +1,16 @@
 import React from "react";
 import { Link } from "react-router-dom";
-import Logo from "../shared/Logo/logo";
+import { IoMdSend } from "react-icons/io";
+import LogoWhite from "../../assets/icons/LogoWhite.png";
 import "./footer.scss";
 import constants from "../../constants/Footer";
 
-const { SUBSCRIBE_US, SUBSCRIBE, CONTACT_US, PHONE_NUMBER } = constants;
+const { SUBSCRIBE_US, CONTACT_US, PHONE_NUMBER, PASTE_YOUR_EMAIL } = constants;
 
 const links = [
   { id: 0, name: "Home", path: "/home" },
```

Comment:
Maybe use specific `id` instead of index of array

Reply:
Done